### PR TITLE
Release notes 2.16.9

### PR DESCRIPTION
**Pull request by serinamarie**
[Preview](https://github.com/PrefectHQ/prefect/blob/release-notes-2.16.9/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,73 @@
...
+In a prior release, `prefect deployment run` added `-jv/--job-variable` for providing job variables. We want to be consistent in our CLI by adding a `prefect deploy` `-jv/--job-variable` option as well! We have deprecated the `-v/--variables` option.
+
+See the following pull request for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/12410
+
+### Enhancements
+- Remove nested task constraint that prevented tasks called from other tasks — https://github.com/PrefectHQ/prefect/pull/12548
+- Stop creating artifacts for unpersisted results - https://github.com/PrefectHQ/prefect/pull/12454
+- Allow for deletion of work pool workers via API — https://github.com/PrefectHQ/prefect/pull/12330
+- Raise nicer error on `prefect worker start -t bad-type` - https://github.com/PrefectHQ/prefect/pull/12586
```
Comment (pleek91) [inline RELEASE-NOTES.md:16]:
```suggestion
- Raise nicer error on `prefect worker start -t bad-type` - https://github.com/PrefectHQ/prefect/pull/12586
- Add tooltip to variable names in the ui and increase width given for displaying variable names https://github.com/PrefectHQ/prefect-ui-library/pull/2275
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,73 @@
...
+
+### Fixes
+- Raise lower bound on `typer` dependency — https://github.com/PrefectHQ/prefect/pull/12512
+- Skip flow run cancellation if no associated deployment — https://github.com/PrefectHQ/prefect/pull/12001
+- Handle referenced blocks in base templates during `job_variable` validation — https://github.com/PrefectHQ/prefect/pull/12329
+- Select correct `AsyncWaiter` for successively awaited flow and task calls — https://github.com/PrefectHQ/prefect/pull/12510
+- Handle flow run creation for runner-managed deployments — https://github.com/PrefectHQ/prefect/pull/12319
+- Expose `ignore_warnings` in `Flow.deploy` — https://github.com/PrefectHQ/prefect/pull/12569
+- Allow `prefect cloud login` re-authentication in non-interactive mode — https://github.com/PrefectHQ/prefect/pull/12575
+- Update ECS provisioner IAM policy to include `ecs:TagResource` permission — https://github.com/PrefectHQ/prefect/pull/12551
```
Comment (pleek91) [inline RELEASE-NOTES.md:26]:
```suggestion
- Update ECS provisioner IAM policy to include `ecs:TagResource` permission — https://github.com/PrefectHQ/prefect/pull/12551
- Correctly populate default parameters in the create flow run form https://github.com/PrefectHQ/prefect-ui-library/pull/2280
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
+- Correctly populate default parameters in the create flow run form https://github.com/PrefectHQ/prefect-ui-library/pull/2280
```
Comment (serinamarie) [inline RELEASE-NOTES.md:28]:
```suggestion
- Correctly populate custom default parameters in the flow submission form https://github.com/PrefectHQ/prefect-ui-library/pull/2280
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,76 @@
...
+In a prior release, `prefect deployment run` added `-jv/--job-variable` for providing job variables. We want to be consistent in our CLI by adding a `prefect deploy` `-jv/--job-variable` option as well! We have deprecated the `-v/--variables` option.
+
+See the following pull request for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/12410
+
+### Enhancements
+- Remove nested task constraint that prevented tasks called from other tasks — https://github.com/PrefectHQ/prefect/pull/12548
+- Stop creating artifacts for unpersisted results - https://github.com/PrefectHQ/prefect/pull/12454
+- Allow for deletion of work pool workers via API — https://github.com/PrefectHQ/prefect/pull/12330
+- Raise nicer error on `prefect worker start -t bad-type` - https://github.com/PrefectHQ/prefect/pull/12586
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:16]:
```suggestion
- Raise more informative error on `prefect worker start -t bad-type` - https://github.com/PrefectHQ/prefect/pull/12586
```

**Review by zzstoatzz — APPROVED** [review]
looks tidy! whew lots of PRs

🙇

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,76 @@
 # Prefect Release Notes
 
+## Release 2.16.9
+
+### `prefect deploy` with `-jv/--job-variable` option
+
+In a prior release, `prefect deployment run` added `-jv/--job-variable` for providing job variables. We want to be consistent in our CLI by adding a `prefect deploy` `-jv/--job-variable` option as well! We have deprecated the `-v/--variables` option.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:7]:
```suggestion
In a prior release, we added a `-jv/--job-variable` for providing job variables when running a deployment via the CLI using `prefect deployment run`. We want to be consistent in our CLI by adding a `-jv/--job-variable` option while creating deployments as well by adding this to our `prefect deploy` command! We have deprecated the `-v/--variables` option.
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,76 @@
 # Prefect Release Notes
 
+## Release 2.16.9
+
+### `prefect deploy` with `-jv/--job-variable` option
+
+In a prior release, we added a `-jv/--job-variable` for providing job variables when running a deployment using `prefect deployment run`. We want to be consistent in our CLI by allowing you to use this option while creating deployments during `prefect deploy`! Thus, we have added a `-jv/--job-variable` option to `prefect deploy` to replace the `-v/--variables` option, which we have now deprecated.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:7]: @zzstoatzz how does this sound?

Comment (serinamarie) [inline RELEASE-NOTES.md:7]:
```suggestion
In a prior release, we added a `-jv/--job-variable` option for providing job variables when running a deployment using `prefect deployment run`. We want to be consistent in our CLI by allowing you to use this option while creating deployments during `prefect deploy`! Thus, we have added a `-jv/--job-variable` option to `prefect deploy` to replace the `-v/--variables` option, which we have now deprecated.
```

Comment (zzstoatzz) [reply RELEASE-NOTES.md]: 💯

**Comment by serinamarie** [comment]
> looks tidy! whew lots of PRs
> 
> 🙇

totally, and we filtered out i think a couple dozen `maintenance` PRs

**Pull request merged**